format-delimited/src/main/java/io/cdap/plugin/format/delimited/common/DataTypeDetectorUtils.java

```diff
@@ -66,7 +66,8 @@ public static void detectDataTypeOfRowValues(Map<String, Schema> override,
    * @return A list of detected schema fields per each column of the dataset.
    */
   public static List<Schema.Field> detectDataTypeOfEachDatasetColumn(Map<String, Schema> override, String[] columnNames,
-    DataTypeDetectorStatusKeeper dataTypeDetectorStatusKeeper) {
+                                                                     DataTypeDetectorStatusKeeper
```

Comment:
Move this to a new line for better legibility

```
public static List<Schema.Field> detectDataTypeOfEachDatasetColumn(
    DataTypeDetectorStatusKeeper dataTypeDetectorStatusKeeper) {
```

Reply:
Yeah, this was an ide issue that I missed. Fixed it now